### ISSUE TITLE
Is there a way to remove the fps cap/stuttering but still retain the correct physics speed?

### ISSUE THREAD

**Issue by Xenphor** [dhewm/dhewm3#94]
As I understand, doom 3 was capped at 60 fps which by itself is not bad but it introduces a horrible stutter effect, especially noticeable with vsync on. There was a way to remove this in the original noted here:

"com_fixedTic [-1,0,1] - If set to 0 (the default) Doom 3's ticrate is capped at 60FPS, meaning the game framerate cannot exceed this, except when running a timedemo. If you want to remove this cap, set com_fixedTic to -1. Setting it to 1 removes the cap, but causes the game physics to go out of sync. Removing the FPS cap doesn't necessarily improve performance, and can lead to more variable framerates. Note that com_fixedTic -1 doesn't work as of the latest patch."

So the problem is if you remove the cap in the latest version, it causes the physics engine to go out of sync if the framerate is not stable. Is there a way to modify this so you can remove the limit but not affect the physics? 

Basically it would be nice to have an option for lower end platforms (or any platforms) to be able to use vsync without the stuttering the engine causes by allowing a graceful drop from 60fps.

**Comment by DanielGibson** [comment]
`com_fixedTic -1` is supported now and I've been told that it doesn't cause trouble

**Comment by DanielGibson** [comment]
(BTW, turned out `com_fixedTic -1` does cause trouble after all)
